### DISCUSSION TITLE
Any maintainer left?

### DISCUSSION THREAD

**Discussion by chbiel** [gpbl/react-day-picker#1089] [General]
It seems there is no maintainer left for this project. Is that correct? So no one left with permissions to apply changes, approve PR's etc.?

**Comment by alexkhismatulin** [comment]
@gpbl hey! Can you clarify the current state of the project, please?

**Comment by kelvinlouis** [comment]
I guess it is save to say that this library is no longer being maintained. This impacts a lot of companies and other open source projects such as blueprint. We (not affiliated with blueprint) are currently refactoring our library and are considering moving away from react-day-picker. It is unfortunate, but there quite a lot of issues that need to be addressed, especially regarding accessibility. I have to say though, it is quite difficult to find a similar or better alternative to `react-day-picker`, particularly with a minimal set of dependencies.

**Comment by oliviertassinari** [comment]
@kelvinlouis How do you plan to solve the date picker problem?

**Comment by kelvinlouis** [comment]
Hey @oliviertassinari,

We are currently evaluating alternatives. No decision has been made yet. Do you have any recommendations?

**Comment by connor-baer** [comment]
@gpbl Would you be open to adding a co-maintainer or transferring ownership of the project? The company I work for ([@sumup-oss](https://github.com/sumup-oss/)) might be willing to provide stewardship of the project.

**Comment by brianespinosa** [comment]
@gpbl I think the suggestion by @connor-baer is a good one. If we can at least get some insight as to the plans for the next version that would be great. It looks like progress is pretty far along. Would you be open to establishing a group of maintainers or orgs interested in taking responsibility for moving things forward?

Also, thank you for your work putting this project together. There have been a lot of mediocre date pickers for React, most of them just clones of a Bootstrap picker. This project IMO has been the best implementation in terms of flexibility and control for multiple use cases with a lot of thought put into the APIs and use cases. I'd hate to see it die because you alone simply don't have the bandwidth to invest at this time.

**Comment by wesm87** [comment]
I'd also be willing to invest some time into updating / maintaining the project. My company is currently working on a project using this package but it's a bit of a concern that there isn't much active development going on as we tend to support projects for years after initial development. We also tried looking at alternatives and found them lacking in terms of flexibility, forcing you to use huge libraries like Moment, being difficult to style or customize how they're rendered, etc. It looks like there's at least [a roadmap in place](https://github.com/gpbl/react-day-picker/projects/7) for the next version so we could just divide-and-conquer, one issue at a time.

**Comment by brianespinosa** [comment]
@wesm87 a lot of that roadmap is already done if you take a look at the working branch for that release. The docs are pretty far along too. IMO it is a significant improvement over the current version of the picker as it already is. That is what I am hoping we can get polished up and out the door if we can get some feedback or clarity from @gpbl. If there are plans to come back to this, I am fine waiting.

**Comment by gpbl** [comment]
Hi everybody, sorry for the absence due to the pandemic (long story). 

I'm merging https://github.com/gpbl/react-day-picker/pull/1086 with lot of changes, so DayPicker is ready for a beta release! I don't know yet about v7, I need to checkout the issues first. 

@connor-baer @wesm87 Thanks a lot for offering your help, I appreciate it! Give me few days to take the project in hand again. And thanks everybody for this thread, it's nice to know a lot of people are using DayPicker.

More updates coming soon!

PS. There's also the sponsor thing: https://github.com/sponsors/gpbl :)

**Comment by brianespinosa** [comment]
@gpbl glad to hear you are still planning to stay around and steer the ship. This is exciting news. 🎉

**Comment by gpbl** [comment]
New website is live: https://react-day-picker-next.netlify.app
Beta version to be released soon on npm.

(Also, I am converting this issue to a discussion and see how it works)

**Reply by StuckInSprings** [reply]
Really glad to you're making great progress with the ongoing udpates, I took a look through number of datepickers for react and this was the best one. v8 looks super promising. I am very tempted to go with v8 for some work stuff I'm doing but am conciouss that full release of v8 may be a few months away yet. Do you think it's far along enough yet that I'd be ok using the beta version?
